src/ray/gcs/gcs_server/gcs_autoscaler_state_manager.cc

```diff
@@ -415,5 +415,29 @@ void GcsAutoscalerStateManager::HandleDrainNode(
       });
 }
 
+std::string GcsAutoscalerStateManager::DebugString() const {
```

Comment:
Do we want to print logs on the raylet sde so we know which raylet has leaked demands